### ISSUE TITLE
Bind position to a property of objects

### ISSUE THREAD

**Issue by GCorbel** [mharris717/ember-drag-drop#81]
I have a `position` property to objects used in a sortable list. I want to update the position at the end of the drop. All position of the group have to be updated.

I didn't saw this option. Is it possible yet ? I'm ready to contribute if you're interested.

**Comment by dgavey** [comment]
I think I understand what your saying, that instead of the position being updated instantly that you just want the position changes returned at the end.  I think in the roadmap (in my head) to 1.0 that will be possible, but isn't currently.

**Comment by GCorbel** [comment]
No, this not what I want. I want to bind the `position` attribute with the position of the object in the list. Is it clearer ?

**Comment by dgavey** [comment]
No way currently to do that other than by updating your position attributes manually after the drop has ended, by mapping it to the index of the updated array.

**Comment by GCorbel** [comment]
Ok. You can clone this issue if you don't want to implement it.
